vcr/stubs/__init__.py

```diff
@@ -315,6 +315,22 @@ def __init__(self, *args, **kwargs):
         with force_reset():
             self.real_connection = self._baseclass(*args, **kwargs)
 
+    def __setattr__(self, name, value):
+        """
+        We need to define this because any attributes that are set on the
+        VCRConnection need to be propogated to the real connection.
+
+        For example, urllib3 will set certain attributes on the connection,
+        such as 'ssl_version'. These attributes need to get set on the real
+        connection to have the correct and expected behavior.
+        """
+        try:
+            setattr(self.real_connection, name, value)
+        except AttributeError: # raised if real_connection has not been set yet
+            pass
+
+        super(VCRConnection, self).__setattr__(name, value)
```

Comment:
It seems like separately setting the attribute on the two instances is not a great solution. I think it would be better of also proxy getattr access though